pkg/commands/list.go

```diff
@@ -16,6 +16,7 @@ package commands
 
 import (
 	"encoding/json"
+	"github.com/praetorian-inc/trident/pkg/db"
```

Comment:
We should move this import down to the bottom (with a newline between viper and this). The general import structure should be standard library, third party, first party.